kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/blockchain/BlockchainSource.java

```diff
@@ -24,30 +25,37 @@
     /**
      * @return the cached balance.
      */
-    int getBalance();
+    BalanceUpdate getBalance();
 
     /**
      * Get balance from network
      * @param callback
      */
-    void getBalance(@NonNull final Callback<Integer> callback);
+    void getBalance(@NonNull final Callback<BalanceUpdate> callback);
 
     /**
-     * Add balance observer in order to listen for updates
+     * Add balance observer in order to start receive balance updates
      * @param observer
      */
-    void addBalanceObserver(@NonNull final Observer<Integer> observer);
+    void addBalanceObserver(@NonNull final Observer<BalanceUpdate> observer);
```

Comment:
no need to add addBalanceObserver and  removeBalanceObserver to BlockchainSource.java interface, right?

Reply:
no, we do we are using it.